### PR TITLE
DBAASCLI-68 Add LoadBalancer detection

### PR DESCRIPTION
**Pull request by nonemax**

### CODE REVIEW

**Comment by JNKPercona** [comment]
License check is ok.

File: dbaas-lib/k8s/cmd.go [hunk 1]
```diff
@@ -52,6 +52,19 @@ type ErrCmdRun struct {
...
+
+type ServiceStatus struct {
+	LoadBalancer struct {
+		Ingress []struct {
+			IP       string `json:"ip"`
+			Hostname string `json:"hostname"`
+		} `json:"ingress"`
+	} `json:"loadBalancer"`
+}
+
```
Comment (dAdAbird) [inline dbaas-lib/k8s/cmd.go]: I don't see any usage of this types

**Review by dAdAbird — CHANGES_REQUESTED** [review]

File: dbaas-lib/k8s/cmd.go (hunk 1 above)
Comment (nonemax) [reply dbaas-lib/k8s/cmd.go]: https://github.com/Percona-Lab/percona-dbaas-cli/blob/DBAASCLI-68/dbaas-lib/engines/k8s-pxc/controller.go#L119

Comment (dAdAbird) [reply dbaas-lib/k8s/cmd.go]: Why not use https://pkg.go.dev/k8s.io/api/core/v1?tab=doc#ServiceStatus ?

Comment (nonemax) [reply dbaas-lib/k8s/cmd.go]: done in https://github.com/Percona-Lab/percona-dbaas-cli/pull/102/commits/7dfd4afa3d703237ee815bc948e147cf1fb48dad

**Review by dAdAbird — APPROVED** [review]

**Comment by JNKPercona** [comment]
License check is ok.

**Pull request merged**